app/aws/xray_celery_handlers.py

```diff
@@ -61,7 +61,7 @@ def xray_task_prerun(task_id=None, task=None, args=None, **kwargs):
 
 
 def xray_task_postrun(task_id=None, task=None, args=None, **kwargs):
-    logger.info(f"xray-celery: postrun: kwargs={kwargs}")
+    logger.debug(f"xray-celery: postrun: kwargs={kwargs}")
```

Comment:
oh yes good idea, I've often been adding a filter to get rid of these in my query 💯 
Though do we still need these logs at all?